images/Containerfile.pipeline

```diff
@@ -10,7 +10,7 @@ RUN curl -k -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/
     curl -k -s https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest/opm-linux.tar.gz | tar xvz  -C /usr/bin && \
     chmod +x /usr/bin/oc /usr/bin/opm /usr/bin/kubectl  /usr/bin/jq  /usr/bin/yq
 
-RUN dnf install -y bind-utils openssh-clients httpd-tools conmon skopeo podman gettext fuse-overlayfs iputils nmap-ncat --setopt=install_weak_deps=False  && \
+RUN dnf install -y bind-utils openssl openssh-clients httpd-tools conmon skopeo podman gettext fuse-overlayfs iputils nmap-ncat --setopt=install_weak_deps=False  && \
```

Comment:
This needs an urgent rebase as iirc the Container.pipeline includes now a couple more rpm.